src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.X509Name.cs

```diff
@@ -13,6 +14,22 @@ internal static partial class Crypto
         [DllImport(Libraries.CryptoNative)]
         internal static extern int GetX509NameStackFieldCount(SafeSharedX509NameStackHandle sk);
 
+        [DllImport(Libraries.CryptoNative)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        internal static extern bool PushX509NameStackField(SafeX509NameStackHandle stack, SafeX509NameHandle x509_Name);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern void RecursiveFreeX509NameStack(IntPtr stack);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern SafeX509NameStackHandle NewX509NameStack();
+
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern int FindX509NameInStack(SafeX509NameStackHandle stack, SafeX509NameHandle x509_Name);
```

Comment:
I don't think you're using FindX509NameInStack any longer; if not, you shouldn't add it here (or in the native library)